### DISCUSSION TITLE
F2 to rename

### DISCUSSION THREAD

**Discussion by Praful** [jarun/nnn#1913] [Ideas]
I noticed that F2 is not bound to anything in nnn. I've compiled a version where F2 triggers a rename, which is what it does on some other file managers, such as nemo (Linux) and File Explorer (Windows).

Could this be considered as a default binding? Is there any downside to adding this shortcut to nnn?

**Comment by jarun** [comment]
I remember it was there at some point but we dropped it since no other function keys are used.

**Reply by jarun** [reply]
You can easily patch and add it though.

**Reply by Praful** [reply]
Thanks, that is what I have done.
